README.md

```diff
@@ -13,8 +13,9 @@ GraphQL endpoint providing a single endpoint to access data from Argo's many ser
 
 - Set up environment: copy `.env.schema` to `.env` and update environment accordingly if needed. Values provided in the schema file can be used when running the server locally for development.
 - Install dependencies: `npm ci` (depending on your Node version, it may require a '--legacy-peer-deps' flag)
-- Navigate to `./compose` and run `docker-compose up -d` to start up elasticsearch
-- `npm run programDashboardEsInit && npm run fileCentricEsInit` to initialize some data to elasticsearch for local development
+- Insure your system has Docker and Postgres installed and running.
+- Navigate to `./compose` and run `docker-compose up -d` to start up elasticsearch & program service
+- Navigate back to project root and run `npm run programDashboardEsInit && npm run fileCentricEsInit` to initialize some data for local development.
 - Run server locally: `npm run dev`
```

Comment:
- There may be a need for additional steps to populate program database:
`docker cp ./ps_dump.sql compose-program-db-1:/ps_dump.sql`
` docker exec -u postgres compose-program-db-1 psql program-db postgres -f /ps_dump.sql`
docker exec line is breaking for me, need to provide `ps_dump.sql` file